baton/hub.go

```diff
@@ -2,9 +2,12 @@ package baton
 
 import (
 	"encoding/json"
-	"log"
+
+	"github.com/Sirupsen/logrus"
 )
 
+var log = logrus.WithField("module", "Hub")
```

Comment:
It is used as a global variable in their examples and it seems to be common. [Here](https://github.com/dolaterio/dolaterio/blob/56b16e47015e4b90b764f47045a8d85aeffe8669/docker/base.go) is an example of it being used as a global variable (for the exact same reason).